### PR TITLE
Feature/achievements

### PR DESCRIPTION
**Pull request by babelarr**
Ready to merge!

### CODE REVIEW

File: src/utils/template.js [hunk 1]
```diff
@@ -51,15 +51,15 @@ function fillDashboardTemplate(viewedItems) {
   // Summary toggle
   let summaryPageToggle = document.querySelector(summaryPageToggleSelector);
   summaryPageToggle.textContent = chrome.i18n.getMessage('summary');
-  let summaryPageToggleIcon = createIconNode('list');
-  summaryPageToggle.prepend(summaryPageToggleIcon);
+  //let summaryPageToggleIcon = createIconNode('list');
```
Comment (hmartos) [inline src/utils/template.js]: It's definitely better to do this with CSS, thanks!. 
This commented lines of code will be removed before merging in the master branch

File: src/utils/template.js [hunk 2]
```diff
@@ -99,12 +99,14 @@ function showStatsSection() {
 
   document.querySelector('#moviesCount .ns-title').textContent = chrome.i18n.getMessage('moviesCount');
   document.querySelector('#moviesCount .ns-title').setAttribute('aria-label', chrome.i18n.getMessage('moviesCount'));
+  document.querySelector('#moviesCount .ns-image').setAttribute('src', chrome.runtime.getURL("../images/movie.svg"));
```
Comment (hmartos) [inline src/utils/template.js]: As the [docs ](https://developer.chrome.com/extensions/runtime#method-getURL) say, the path inside _getURL_ function should be relative to the extension install directory, so I will change this path and the other four

**Comment by hmartos** [comment]
Merged in the feature/achievements branch. The new icons and the new logo are amazing, thanks!

File: src/utils/template.js (hunk 1 above)
Comment (babelarr) [reply src/utils/template.js]: Perfect! thanks!

File: src/utils/template.js (hunk 2 above)
Comment (babelarr) [reply src/utils/template.js]: ok, no problem ;)

**Comment by babelarr** [comment]
Thank you very much for your corrections. It's a pleasure to work with you :)

**Comment by hmartos** [comment]
Thank you very much for your work

**Pull request merged**